lib_xua/api/xua_conf_default.h

```diff
@@ -1646,3 +1650,13 @@ enum USBEndpointNumber_Out
 #ifndef WINUSB_DEVICE_INTERFACE_GUID
 #define WINUSB_DEVICE_INTERFACE_GUID               "{89C14132-D389-4FF7-944E-2E33379BB59D}"
 #endif
+
+/**
+ * @brief bRequest of the D2H vendor request that the host will send to read the MSOS descriptor
+ *
+ * The user shouldn't use this bRequest number in their own custom vendor request to
+ * the device. Override in application if necessary.
+ */
+#ifndef REQUEST_GET_MS_DESCRIPTOR
+#define REQUEST_GET_MS_DESCRIPTOR   0x20
```

Comment:
Not sure this needs to be over-ridable (and in xua_conf_default)? We don't want users to change this, right?

Reply:
Hmm, my thinking was that if the user has a host application that uses bRequest 0x20 then they could change the REQUEST_GET_MS_DESCRIPTOR to avoid a conflict. I guess they could as well change the host app..

Reply:
I've moved it back to xua_ep0_msos_descriptors.h